src/utils/toInt.js

```diff
@@ -14,6 +14,6 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
-const toInt = (string) => parseInt(string, 10);
+const toInt = (string) => parseFloat(string, 10);
```

Comment:
If you look at the name of the functions expected output is an integer, not float.
Why do you need this change?

Reply:
Because in `value` of `getNormalizedStateData` could be floats (aka percentage numbers). I could create new `toFloat` function and use it on `value`.